src/pybind/mgr/cli_api_commands/module.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, func: Callable):
         self.signature = inspect.signature(func)
         self.params = self.signature.parameters
 
-    def to_ceph_signature(self):
+    def to_ceph_signature(self) -> Any:
```

Comment:
Try to use specific types instead of using Any as a wildcard.
```suggestion
    def to_ceph_signature(self) -> Dict[str, str]:
```